style: apply /style-guide pass to models/track (#2728)
wandb/docs: models/track/log/plots.mdx

title: Create and track plots from experiments
---

> This page shows you how to create and log plots from your machine learning experiments so that you can visualize metrics, evaluate models, and share interactive charts in W&B. This page is for you if you already log runs with the W&B Python SDK (`wandb`) and want to add custom or built-in visualizations to your experiments.

Reviewer: ```suggestion
This page shows you how to create and log plots from your machine learning experiments so that you can visualize metrics, evaluate models, and share interactive charts in W&B. This page assumes you already log runs with the W&B Python SDK (`wandb`) and want to add custom or built-in visualizations to your experiments.
```

Author: Applied your suggestion in `cf5b4357`.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_

> In W&B Models, methods in `wandb.plot` let you track charts with `wandb.Run.log()`, including charts that change over time during training. To learn more about the custom charting framework, see the [custom charts walkthrough](/models/app/features/custom-charts/walkthrough/).

Reviewer: ```suggestion
Use functions from W&B's `wandb.plot` namespace to track charts. To learn more about the custom charting framework, see the [custom charts walkthrough](/models/app/features/custom-charts/walkthrough/).
```

^Then link to wandb.plot  reference

Author: Applied your suggestion in `cf5b4357`.

_🤖 Addressed by [Claude Code](https://claude.com/claude-code)_
